packages/showcases/core/csf/components/overlays/popover.csf.js

```diff
@@ -7,3 +7,39 @@ export const parameters = {
     url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2756%3A10788',
   },
 };
+
+export const argTypes = {
+  position: {
+    control: 'select',
+    options: [
+      'top-left',
+      'top',
+      'top-right',
+      'right',
+      'bottom-right',
+      'bottom',
+      'bottom-left',
+      'left',
+    ],
+    description: 'The position of the popover.',
+    defaultValue: 'bottom',
```

Comment:
```suggestion
    defaultValue: 'top',
```

Isn't it top by default ?